cmake/dependencies.cmake

```diff
@@ -628,27 +628,22 @@ IF(ENABLE_LIBXML2)
 ENDIF(ENABLE_LIBXML2)
 
 ################################
-# MPI
+# parallel IO
 ################################ 
-IF(ENABLE_PARALLEL4 OR HDF5_PARALLEL)
-  FIND_PACKAGE(MPI REQUIRED)
+IF(ENABLE_PNETCDF)
+  find_package(PNETCDF 1.6.0 REQUIRED)
 ENDIF()
 
 ################################
-# parallel IO
-################################ 
-IF(ENABLE_PNETCDF)
-  FIND_LIBRARY(PNETCDF NAMES pnetcdf)
-  FIND_PATH(PNETCDF_INCLUDE_DIR pnetcdf.h)
-  IF(NOT PNETCDF)
-    MESSAGE(STATUS "Cannot find PnetCDF library. Disabling PnetCDF support.")
-    SET(USE_PNETCDF OFF CACHE BOOL "")
-  ENDIF()
+# MPI
+################################
+IF(ENABLE_PARALLEL4 OR HDF5_PARALLEL OR ENABLE_PNETCDF)
+  FIND_PACKAGE(MPI REQUIRED)
```

Comment:
These should be handled by the dependencies and not by the consumer. For starters, copy this to `FindPNETCDF.cmake` as well.